Projects/12-Inferential_Stats_P01/Inferential_Stats_P01.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+from scipy.stats import chi2_contingency
+from scipy.stats import fisher_exact
+import matplotlib as plt
+
+# load data (already cleaned)
+df = pd.read_csv('ISPY_clinical_clean.csv');
+df.dropna(axis=0, inplace=True)
```

Comment:
http://stackoverflow.com/questions/22532302/pandas-peculiar-performance-drop-for-inplace-rename-after-dropna